### DISCUSSION TITLE
add digital certificate

### DISCUSSION THREAD

**Discussion by bsouza-site** [netbox-community/netbox#12010] [Help Wanted!] [answered]
Hello!

I followed all the Netbox documentation and performed the installation normally.
However, I received a signed digital certificate (I received the .crt and .key files).

I added the certificates in the directories below and changed the configuration in /etc/httpd/conf.d/apache.conf according to the example below.

![2](https://user-images.githubusercontent.com/47142683/226332817-3d6a4df5-4301-4448-83c0-bbe1224a3bbe.jpg)

However, after that, I can't start Apache anymore. **What is the correct way to add these certificates to Netbox?**

Below, the error that Apache presents.
Job for httpd.service failed because the control process exited with error code.
See "systemctl status httpd.service" and "journalctl -xeu httpd.service" for details.

**Comment by candlerb** [comment, marked as answer]
This isn't really a Netbox question, but first you need to see the actual error: run the command `journalctl -eu httpd.service` as it already suggested. (I omit the -x because it makes the output noisier)

When we see this, then the problem may be clear - e.g. a permissions problem.  If the permissions look right, then maybe it's an SELinux problem. (Since you say "httpd" rather than "apache2" then I guess you're using a RedHat-derived distribution)

Finally: please don't post screenshot images, please just copy-paste the text. It's *much* easier to read.

**Comment by bsouza-site** [comment]
Hi brian.

I thank you for the help. The problem was really in SELinux.
